Translations/fr.lproj/InfoPlist.strings

```diff
@@ -57,7 +57,7 @@
 "Safari extension" = "Extension Safari";
 "ACE Archive" = "Archive ACE";
 "XIP Secure Archive" = "Archive sécurisée XIP";
-"Electronic Book" = "Electronic Book";
+"Electronic Book" = "Livre électronique";
 "Apple Archive" = "Apple Archive";
```

Comment:
This one is translated in the bundled macOS extractor tool so might as well translate it 😛